Formula/coala.rb

```diff
@@ -0,0 +1,56 @@
+class Coala < Formula
+  include Language::Python::Virtualenv
+
+  desc "provides a command-line interface for linting and fixing all your code."
+  homepage "http://coala.io/"
+  url "https://github.com/coala/coala/archive/0.8.1.tar.gz"
+  version "0.8"
+  sha256 "23ece17d2ba0adc147f60b570960a06fe70eef465dead4e1fc95e81e3a8071c1"
+  head "https://github.com/coala/coala.git"
+
+  depends_on :python3
+  resource "appdirs" do
```

Comment:
Could you add a blank line above here, and between each resource? Makes reading the formula easier.